bundles/org.openhab.binding.ipcamera/README.md

```diff
@@ -0,0 +1,1312 @@
+
+**externalMotion**
+
+This channel is intended to allow any external sensor the ability to inform the binding that there is motion in the cameras field of view. 
+Note: It will not be passed onto your camera and will not trigger any recordings, it is purely for the bindings own set of features. 
+An example of how this is useful is if your camera either has no motion alarm features, or you have bugs tripping the built in sensors, you can use this channel with a ZWave PIR sensor to inform the camera of motion.
+This becomes more important when you start to use Camera groups that dynamically change the display order of the cameras based on if there is movement or not.
+It can also be handy to use this when doing testing as it allows motion to be simulated with the press of a button. 
+
+**updateImageNow**
+
+This control can be used to manually start and stop updating the Image channel with a picture, or it will start and stop FFmpeg from creating snapshots from a RTSP source depending on how the bindings config parameters are set. 
+The `UPDATE_IMAGE` config sets the state this control is set to on startup/reboot of Openhab.
+When ON the image channel will update at the `POLL_CAMERA_MS` rate. 
+Note that cameras that create snapshots from RTSP using FFmpeg will not update the image channel at all and the better methods like ipcamera.jpg covered in this readme, are the recommended way to achieve a picture. 
+When OFF the Image channel will NOT update, but the other methods of achieving a picture or stream will still work. 
+If you need to update the image channel more often then every 5 seconds, please see the snapshot and stream sections of this readme to learn how to get a picture without using the Image channel. 
+
+**ffmpegMotionControl**
+
+This control allows FFmpeg to detect movement from a RTSP or HTTP source and inform Openhab. The higher the number, the less sensitive the camera is to movement or you have to wave your hand faster to trigger the alarm.
+It is best described in the first few posts of this thread, along with how to fault find it using a terminal/command line.
+<https://community.openhab.org/t/how-to-turn-a-cameras-rtsp-stream-into-motion-detection/89906>
+You can link a Switch and a Slider to this channel at the same time to have both ON/OFF switch control, as well as a slider to change the threshold.
+The channel that will move is called ``ffmpegMotionAlarm``.
+
+**thresholdAudioAlarm**
+
+Most of the API cameras have a separate ON/OFF channel, but for non API cameras that use ffmpeg to create an Audio Alarm from a RTSP source, this channel can be linked to a Switch and a Slider.
+Linking both controls to this channel at the same time gives ON/OFF control as well as a slider to change the threshold. 
+The value of the slider is the value in dB that is detected as no noise/alarm. 
+Higher values are more sensitive and will trigger the Alarm with quieter / less noise.
```

Comment:
Yes, at the time I felt it makes more sense that way, but I do see your point. The value of the slider is the value in Decibels below what is called digital full scale (maximum volume).  A value of 6 on a slider is -6db threshold.... Since I come from an audio background it makes sense to me, but for most people it probably feels back to front.

Reply:
I have Re worked the description to make it clearer in readme.